src/NuGetPackageVerifier/Rules/AssemblyStrongNameRule.cs

```diff
@@ -10,6 +10,7 @@
 using NuGet.Packaging;
 using NuGetPackageVerifier.Logging;
 
+#if NET451
```

Comment:
Apparently we might be able to do this using Mono.Cecil. According to http://thread.gmane.org/gmane.comp.gnome.mono.cecil/3315, we could use their introspection APIs to determine if an assembly is strong named (including determining if it's delay signed). 

If we can

a. empirically prove that this works (try this with non-strong named, delay signed and strong name signed binaries on binaries produced on Windows + xplat)
b. @Eilon is happy with us not using a blessed API for strong name verification

We should do it. It would also mean we wouldn't have to cross compile this binary.